pkg/integrations/manager.go

```diff
@@ -231,18 +238,27 @@ func (m *Manager) instanceConfigForIntegration(cfg Config, i Integration) instan
 	relabelConfigs := append(m.defaultRelabelConfigs, common.RelabelConfigs...)
 
 	var scrapeConfigs []*config.ScrapeConfig
+	schema := "http"
+	// Check for HTTPS support
+	httpClientConfig := config_util.HTTPClientConfig{}
+	if m.c.SeverUsingTLS {
+		schema = "https"
+		httpClientConfig.TLSConfig = m.c.TLSConfig
+	}
+
```

Comment:
Can you move this check above the declaration for `scrapeConfigs`? I tend to keep variables declared along with code that initializes them. Moving this would help keep them together.